app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,11 @@
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
   <color name="ic_launcher_background">#26A69A</color>
+
+  <!--  BASIC COLORS-->
+  <color name="blue">#2D4A9D</color>
```

Comment:
This isn't true blue--could we have a better name for it? Is it one of Oppia's blue colors? If so, maybe something like ``oppia_dark_blue`` would be more fitting.

Reply:
Noted. From now on, oppia-color codes will have `oppia` prefix and other true colors will use named directly.

Reply:
Also, the name should be `oppiaDarkBlue` and not `oppia_dark_blue` based on `colorPrimaryDark` naming convention.